specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the management group."
+        },
+        "details": {
+          "title": "Details",
+          "$ref": "#/definitions/ManagementGroupDetails"
+        },
+        "children": {
+          "description": "The list of children.",
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupChildInfo"
+          }
+        }
+      }
+    },
+    "ManagementGroupDetails": {
+      "description": "The details of a management group.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "number",
+          "format": "int32",
+          "description": "The version number of the object."
+        },
+        "updatedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The date and time when this object was last updated."
+        },
+        "updatedBy": {
+          "type": "string",
+          "description": "The identity of the principal or process that updated the object."
+        },
+        "parent": {
+          "title": "Parent",
+          "$ref": "#/definitions/ParentGroupInfo"
+        },
+        "path": {
+          "description": "The path from the root to the current group.",
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupPathElement"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "managementGroupAncestors": {
+          "description": "The ancestors of the management group.",
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "managementGroupAncestorsChain": {
+          "description": "The ancestors of the management group displayed in reversed order, from immediate parent to the root.",
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupPathElement"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        }
+      }
+    },
+    "ManagementGroupChildInfo": {
+      "description": "The child information of a management group.",
+      "properties": {
+        "type": {
+          "title": "The type of child resource.",
+          "$ref": "#/definitions/ManagementGroupChildType",
+          "description": "The fully qualified resource type which includes provider namespace (e.g. Microsoft.Management/managementGroups)"
+        },
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the child resource (management group or subscription).  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000"
+        },
+        "name": {
```

Comment:
>name [](http://example.com/codeflow?start=9&length=4)

Is this redundant since you already have id/resourceId? Note: property names within the property bag are not allowed to duplicate top-level property names. Recommend providing either resourceId or managementGroupName here.